modules/openy_features/openy_media/modules/openy_media_image/openy_media_image.install

```diff
@@ -328,7 +328,7 @@ function openy_media_image_update_8013() {
  *
  * Blazy image styles don't work with image alignment in CKEditor, so we shouldn't use them.
  */
-function openy_media_image_update_8014() {
+function openy_media_image_update_8015() {
```

Comment:
I don't think we need this change. The only situation these configs would need fixing is when you install the current Open Y - but this installation is broken currently.
If any site has already applied the 8014 - then it doesn't need this 8015.

Reply:
Fixed